geoindicators/src/main/groovy/org/orbisgis/RsuIndicators.groovy

```diff
@@ -176,13 +185,17 @@ static IProcess groundSkyViewFactor() {
                         "ST_DWITHIN(b.$geometricColumnBu, a.the_geom, $rayLength) GROUP BY a.the_geom").toString())
 
                 // The result of the SVF calculation is averaged at RSU scale and the rsu that do not have
-                // buildings in the area of calculation are considered "free sky" (SV = 1)
+                // buildings in the area of calculation are considered "free sky" (SVF = 1)
                 datasource.execute(("CREATE INDEX IF NOT EXISTS id_svf ON $svfPts(id_rsu); " +
                         "CREATE INDEX IF NOT EXISTS id_pts ON $ptsRSUtot(id_rsu);"+
                         "DROP TABLE IF EXISTS $outputTableName; CREATE TABLE $outputTableName AS "+
                         "SELECT COALESCE(AVG(a.SVF),1.0) AS rsu_ground_sky_view_factor, b.id_rsu AS $idColumnRsu " +
                         "FROM $svfPts a RIGHT JOIN $ptsRSUtot b ON a.id_rsu = b.id_rsu "+
                         "GROUP BY b.id_rsu").toString())
+                def tObis = System.currentTimeMillis()-to_start
+
+                println "OrbisGIS calculation time: ${tObis/1000} s"
```

Comment:
instead of println you can use LOGGER.info(String) or LOGGER.warn(String)

Reply:
done